example/spec/helpers/Collections.js

```diff
@@ -0,0 +1,86 @@
+const { listGranules, removePublishedGranule, deleteGranule } = require('@cumulus/api-client/granules');
+const { getPdrs, deletePdr } = require('@cumulus/api-client/pdrs');
+const { deleteExecution, getExecutions } = require('@cumulus/api-client/executions');
+const { deleteCollection } = require('@cumulus/api-client/collections');
+const { constructCollectionId } = require('@cumulus/message/Collections');
+/**
+* Helper to remove a collection and all it's dependencies
+* @summary Uses api-client to search for collection dependencies, remove them all and then remove the collection
+*          Cleans up:
+*            - published/unpublished Granules
+*            - PDRs
+*            - Executions
+*            - the specified Collection
+* @param {Object} params     - params
+* @param {string} prefix  - Config object containing stackName
+* @param {Object} collection - Cumulus API collection object to delete
+* @return {Promise<undefined>}
+*/
+const removeCollectionAndAllDependencies = async (params) => {
+  const { prefix, collection } = params;
```

Comment:
I admit that I'm not a huge fan of complex helper functions like this that do a lot of things vs integration tests making sure they are precise in their cleanup. But I could probably be convinced